docs/topics/forms/formsets.txt

```diff
@@ -150,7 +150,7 @@ protects against memory exhaustion attacks using forged ``POST`` requests::
     >>> formset.is_valid()
     False
     >>> formset.non_form_errors()
-    ['Please submit at most 1000 forms.']
+    ['Please submit at most 1500 forms.']
```

Comment:
Have you check this example? Message is correct.

Reply:
Hi @felixxm, I see, thanks.

It looks to me that the message is misleading.

Consider a simple example:
```
ArticleFormSet = formset_factory(ArticleForm, max_num=2, absolute_max=4)
```

For 4 forms, the formset is valid, no errors.
For 5 forms, the formset is not valid, non form errors tells you to submit at most 2 forms, when 4 is working properly.

In this case: https://github.com/django/django/blob/master/django/forms/formsets.py#L368 only `absolute_max` is checked, so it should be used in the ValidationError, but `max_num` is always used.


Just my two cents, not really interested in discussing it as it is a really small detail.